packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -34,9 +35,8 @@ export class RetriableDocumentStorageService implements IDocumentStorageService
         return this.internalStorageService.policies;
     }
 
-    public dispose() {
-        this.disposed = true;
-    }
+    public get disposed() {return this.runDisposedOnce.evaluated;}
+    public readonly dispose = ()=>this.runDisposedOnce.value;
```

Comment:
I find earlier logic easier to follow (and cheaper :))

Reply:
reverted